frontend/src/static/js/v_authorship.js

```diff
@@ -165,12 +184,21 @@ window.vAuthorship = {
           }), {});
     },
 
+    filesSort() {
+      this.selectedFiles.sort(window.comparator(filesSortDict[this.filesSortType]));
+
+      if (this.filesSortReverse) {
```

Comment:
same as the above.
name it as reverseSortFiles. 
Also, to comply with our coding convention, the boolean variables/methods should be named to sound like booleans.
E.g. prefixed with `is`, `can`, `should`, `to` etc. 
For this case, let's name it `toReverseSortFiles`.